src/brev/produserbaredokumenter.js

```diff
@@ -25,7 +25,7 @@ const produserbaredokumenter = [
   },
   {
     kode: 'IKKE_YRKESAKTIV_VEDTAKSBREV',
-    term: 'Vedtak om medlemskap'
+    term: 'Vedtak om pliktig medlemskap'
```

Comment:
Hvorfor?

Reply:
Fag ønsker å gjøre disse titlene like for de vedtakene som er pliktige. IKKE_YRKESAKTIV_VEDTAKSBREV gjelder ikke-yrkesaktiv EØS og trygdeavtaler, alle bestemmelsene innenfor IKKE_YRKESAKTIV_VEDTAKSBREV resulterer i pliktig medlemskap.